### ISSUE TITLE
New Device CE Power Strip

### ISSUE THREAD

**Issue by darrenblanchard** [jdeath/Hubspace-Homeassistant#24]
I had to add the following to my light.py to get access to each outlet on the strip.

        elif model == 'LTS-4G-W':
            _LOGGER.debug("Creating Outlets" )
            entities.append(HubspaceOutlet(hs, friendlyName,"1",debug))
            entities.append(HubspaceOutlet(hs, friendlyName,"2",debug))
            entities.append(HubspaceOutlet(hs, friendlyName,"3",debug))
            entities.append(HubspaceOutlet(hs, friendlyName,"4",debug))
            _LOGGER.debug("creating lights" )
            entities.append(HubspaceLight(hs, friendlyName,debug))

I am at version 1.71, This is a 4-outlet power strip, and I'm also able to turn off the entire strip separate from each outlet.

Thanks

Darren

**Comment by jdeath** [comment]
Added with [8513e93](https://github.com/jdeath/Hubspace-Homeassistant/commit/8513e9396bd4f69ae20c8f232689caf39b05b4c1)

**Comment by darrenblanchard** [comment]
[TestHubspaceOutput.txt](https://github.com/jdeath/Hubspace-Homeassistant/files/10199861/TestHubspaceOutput.txt)

here is output, this device has 4 outlets and the ability to turn off/on the entire strip from one button if wanted. So, I get an entity for the strip and 4 entities one for each outlet.

Darren

**Comment by darrenblanchard** [comment]
I used the code from the Transformers to make it work for me, the 5th entity worked as a global switch even before the code above was added, so I just took the line from the "Else" tacked to the model number code and it worked.  You are correct the 5th entity in the above code will turn off/on all 4 outlets global switch. Essentially the 5th entity is the strip itself.

The link to the model is correct.

If you would like me to test the 5th entity as an outlet instead of light I can, or if the syntax to that 5th entity should be something different let me know.
